.github/workflows/terraform-observe_scheduler.yaml

```diff
@@ -11,7 +11,7 @@ on:
         description: "Terraform version"
         required: false
         type: string
-        default: "1.2.9"
+        default: "1.3.6"
```

Comment:
thanks @bendrucker  ! do you prefer i just leave it to `latest` instead? 

I basically want to make sure we're not testing stuff that works in CI but breaks in terraformer, which uses 1.3.6. 

If you agree with the reason to go with `1.3.6`, then i'll add a comment explaining that, good idea.

Reply:
Oh 1.3.6 is fine, just cite where it came from (i.e. a comment with a URL that points to the source file/line). Presumably 1.2.9 was selected with the same motivation and so I'm just looking to avoid a repeat when these inevitably get out of sync again. Probably worth linking here from the monorepo too.

Reply:
I put some notes in Slack, in theory we should give modules control over this via https://developer.hashicorp.com/terraform/language/settings#specifying-a-required-terraform-version. Loosening this coupling gets very complex very quickly so for now it definitely makes sense to stick with documenting why/to what the version is pinned.

Reply:
thanks for this! after further slack conversation, i updated to `latest` - but let me know if i've missed anything still